src/bin/pgcopydb/catalog.c

```diff
@@ -7170,6 +7222,113 @@ catalog_iter_s_table_in_copy_init(SourceTableIterator *iter)
 }
 
 
+/*
+ * catalog_iter_s_index_not_processed iterates over the list of indexes which
+ * are not processed in our catalogs. This might be due to user interruption.
+ */
+bool
+catalog_iter_s_index_not_processed(DatabaseCatalog *catalog,
+								   void *context,
+								   SourceIndexIterFun *callback)
+{
```

Comment:
Naming: I'd rather have “remaining_indexes” compared to “index_not_processed”. Maybe “indexes_to_build”?